README.md

```diff
@@ -107,18 +71,63 @@ exports.handler = function(context, event, callback) {
     callback(null, response);
 };
 ```
-5. Save the function
-6. Open [Configure](https://www.twilio.com/console/functions/configure) page and setup values for the following `Environment Variables`:
-    - SERVICE_SID
-        - Open [Conversational Messaging](https://www.twilio.com/console/conversations/configuration/defaults)
-        - Select `View Service` near the `Default Conversation Service`
-        - Copy the `Service SID`
-        - Also navigate to `Push configuration` and enable all types of notifications for receiving FCM messages
-    - TWILIO_API_KEY and TWILIO_API_SECRET
-        - Create an API KEY [here](https://www.twilio.com/console/chat/project/api-keys)
-    - PUSH_CREDENTIAL_SID
-        - This requires Firebase cloud messaging to be configured and is optional, without it push notifications will not be sent
-        - Create new push credentials [here](https://www.twilio.com/console/conversations/push-credentials) using `Server key` from `Firebase Cloud Messaging`
+5. Save the function.
+6. Open the **Environment Variables** tab from the **Settings** section and: 
+    - Check the **"Add my Twilio Credentials (ACCOUNT_SID) and (AUTH_TOKEN) to ENV"** box, so that you get `ACCOUNT_SID` automatically.
+    - Add SERVICE_SID
+        - Open [Conversations Services](https://www.twilio.com/console/conversations/services)
+        - Copy the `SID` for `Default Conversations Service`, or the service you want to set up.
+    - Add `TWILIO_API_KEY_SID` and `TWILIO_API_KEY_SECRET`. Create API Keys [in the console](https://www.twilio.com/console/project/api-keys).
+    - Optionally add `PUSH_CREDENTIAL_SID`, for more info see [Setting up Push Notifications](#setting-up-push-notifications)
+
+### Set the Token Service URL
+
+Set the value of `REACT_APP_ACCESS_TOKEN_SERVICE_URL` in your `.env` file to point to a valid Access-Token server. If an `.env` file doesn't exist, create one with the following contents:
```

Comment:
There's a .env.example file in the repo - it would be simpler to just rename that one and add the url there.